contracts/base/DAIPool.sol

```diff
@@ -43,31 +45,49 @@ contract DAIPool is DAIPoolInterface, Initializable {
 
     LenderInfoInterface public lenderInfo;
 
-    LoansInterface public loanInfo;
+    address public loans;
 
     /** Modifiers */
 
+    /**
+        @notice It checks the address is the Loans contract address.
+        @param anAddress to verify.
+        @dev It throws a require error if parameter is not equal to loans contract address.
+     */
+    modifier isLoan(address anAddress) {
```

Comment:
This modifier is only ever used with the parameter `msg.sender`. The modifier parameter can therefore be removed, and just be` isLoan()` with `loans.requireEqualTo(msg.sender)`. Much like how `onlyOwner` doesn't take a parameter, it just uses msg.sender.